src/util/token.ts

```diff
@@ -0,0 +1,26 @@
+import { Issuer } from "../types";
+import { contractInstance } from "../common/smartContract/contractInstance";
+
+const getContractInstance = (issuers: Issuer[]) => {
+  return issuers
+    .map(issuer => (issuer.tokenRegistry ? contractInstance({ contractAddress: issuer.tokenRegistry }) : null))
+    .filter(instance => !instance);
+};
+
+export const getOwnerOf = async (_tokenId: string, issuers: Issuer[]): Promise<string> => {
+  const contractInstances = getContractInstance(issuers);
+
+  const result = await Promise.all(contractInstances.map(instance => instance.ownerOf(_tokenId)));
+  const owner = result.find(o => o);
```

Comment:
What is this trying to do? Seemed to me to be returning the first element of an array.

Reply:
yea..There would be 1 contract instance only. so find will just make sure to return the result.

Reply:
Can you not use find but use [0]. It's much easier to read that way.